Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:alpine
+
+MAINTAINER atohutchful
+
+RUN apk add --no-cache tini mariadb-client
+
+ADD . /go/src/github.com/siddontang/go-mysql
+
+RUN apk add --no-cache mariadb-client
+
+RUN cd /go/src/github.com/siddontang/go-mysql/ && \
+    go build -o bin/go-mysqldump ./cmd/go-mysqldump && \
+    cp -f ./bin/go-mysqldump /go/bin/go-mysqldump
+
+ENTRYPOINT ["/sbin/tini","--","go-mysqldump"]
```

Comment:
em, I think this may be a dockerfile for all the go-mysql binaries, not only for mysqldump.